### PR TITLE
tidy wetlands tool sections

### PR DESCRIPTION
**Pull request by BexDunn**
move wetlands insight tool sections into a group

### CODE REVIEW

**Comment by pindge** [comment]
has this been tested?

File: dev/terria/dea.json [hunk 1]
```diff
@@ -2599,53 +2599,68 @@
...
-               }
-            },
-            {
-               "name": "DEA WIT QLD Lacustrine v1.0",
-               "url": "https://dea-public-data-dev.s3-ap-southeast-2.amazonaws.com/Wetlands_Insight_Tool/QLD/Lacustrine/Wet_WGS84_L_run_v1.geojson",
-               "type": "geojson",
-               "featureInfoTemplate": {
-                  "template": " <p/> <figure> <img style='width: 100%' src='{{#terria.urlEncode}}http://dea-public-data-dev.s3-ap-southeast-2.amazonaws.com/Wetlands_Insight_Tool/QLD/Lacustrine/{{OBJECTID}}_{{CATCHMENT}}_{{HAB}}.png{{/terria.urlEncode}}'/> <figcaption>Change in water, wet vegetation, green vegetation, dry vegetation, and bare soil. </figcaption> </figure> <p/> Download: <a href='{{#terria.urlEncode}}http://dea-public-data-dev.s3-ap-southeast-2.amazonaws.com/Wetlands_Insight_Tool/QLD/Lacustrine/{{OBJECTID}}_{{CATCHMENT}}_{{HAB}}.png{{/terria.urlEncode}}'>chart image</a> | <a href='{{#terria.urlEncode}}http://dea-public-data-dev.s3-ap-southeast-2.amazonaws.com/Wetlands_Insight_Tool/QLD/Lacustrine/{{OBJECTID}}_{{CATCHMENT}}_{{HAB}}.csv{{/terria.urlEncode}}'>timeseries CSV</a><p/>"
-               }
+	        {
```
Comment (pindge) [inline dev/terria/dea.json:2602]: please align the brackets

**Review by pindge — CHANGES_REQUESTED** [review]

File: dev/terria/dea.json (hunk 1 above)
Comment (BexDunn) [reply dev/terria/dea.json:2602]: oh no! I missed one :( I was trying. Yes, it has been tested using an online JSON parser + pycharm. Is that ok? I'll fix the brackets now

Comment (pindge) [reply dev/terria/dea.json:2602]: not exactly the testing I am thinking of, i can test it for u.

**Comment by pindge** [comment]
@BexDunn 
image

File: dev/terria/dea.json (hunk 1 above)
Comment (BexDunn) [reply dev/terria/dea.json:2602]: Thanks for teaching me how to test this. I have tested the tidied up file on terria and it should be good to go @pindge ?

Comment (pindge) [reply dev/terria/dea.json:2602]: everything else looks good, but this starting `{` is not aligned with the above

**Review by pindge — CHANGES_REQUESTED** [review]

File: dev/terria/dea.json [hunk 2]
```diff
@@ -2599,53 +2599,68 @@
...
+               "name": "DEA Wetlands Insight Tool",
```
Comment (pindge) [inline dev/terria/dea.json:2603]: now i look closely, this line `name` isn't aligned with the rest, please double check

**Review by pindge — CHANGES_REQUESTED** [review]

**Review by pindge — APPROVED** [review]

**Pull request merged**